docs/connector-development/config-based/stream-slicers.md

```diff
@@ -0,0 +1,170 @@
+# Stream Slicers
+
+`StreamSlicer`s define how to partition a stream into a subset of records.
+
+It can be thought of as an iterator over the stream's data, where a `StreamSlice` is the retriever's unit of work.
+
+When a stream is read incrementally, a state message will be output by the connector after reading every slice, which enable checkpointing.
+
+At the beginning of a `read` operation, the `StreamSlicer` will compute the slices to sync given the connection config and the stream's current state,
+As the `Retriever` reads data from the `Source`, the `StreamSlicer` keeps track of the `Stream`'s state, which will be emitted after reading each stream slice.
+
+More information of stream slicing can be found in the [stream-slices section](../cdk-python/stream-slices.md)
+
+## Implementations
+
+This section gives an overview of the stream slicers currently implemented.
+
+### Datetime
+
+The `DatetimeStreamSlicer` iterates over a datetime range by partitioning it into time windows.
```

Comment:
You might want to remind that we are talking about slicing on cursor value

Reply:
updated